docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -54,9 +65,9 @@ The following example shows how to create a gallery-like flexbox layout using th
     flexContainer.Add(toolBar);
     ```
 
-4.  To make the buttons and title display horizontally, but vertically aligned to the center of the toolbar with equal spaces between them, set the toolbar flex direction to **row**, its `AlignItems` property to **center** and `JustifyContent` property to **spaceBetween**:
+5.  To make the buttons and title display horizontally, with vertical alignment to the center of the toolbar and having equal space between them, set the toolbar flex direction to **row**, its `AlignItems` property to **center** and `JustifyContent` property to **spaceBetween**:
```

Comment:
@pkalota Just broke the sentence for better comprehension. 
```suggestion
5.  To make the buttons and title display horizontally, with vertical alignment to the center of the toolbar, and having equal space between them. Set the toolbar flex direction to **row**, its `AlignItems` property to **center**, and `JustifyContent` property to **spaceBetween**:

Reply:
@nawab-samsung Would you please check this suggestion again? It is breaking sentence into uncompleted sentences.
The first sentence becomes weird with this update.

Reply:
@mijin85cho  sure

Reply:
@nawab-samsung maybe it is better to leave comma before `set`? By setting dot there you are cutting first sentence, which started with `To make the buttons and title display horizontally...`, so we should provide explanation how to make it in the same sentence - `, set the toolbar flex direction to **row**...`. In case of your suggestion sentence seems to be uncompleted as @mijin85cho mentioned before.

You are proposing 2 commas before `and` as well. As far as I know, let me know if I'm wrong, in this case both of solutions are correct, but in my mind it looks better without commas. What about leaving it without any change?

Generally it is good idea to split this long sentence. What do you think @newab-samsung and @mijin85cho about following description of this subpoint?

5. To change display of toolbar set the `FlexDirection` to **row**, its `AlignItems` property to **center** and `JustifyContent` property to **spaceBetween**. In result buttons and title will be displayed horizontally, with vertical alignment to the center of the toolbar and having equal space between

Reply:
I think changing the sentence in this way is much better than just cutting the sentence. Adding to that, how about some modification like below?

Change the display of toolbar by setting the `FlexDirection` to **row**, `AlignItems` property to **center**, and **JustifyContent** property to **spaceBetween**. It displays buttons and title horizontally, with vertical alignment to the center of the toolbar and having equal space between them.

Reply:
thanks @pkalota for making me understand this. 

Yes @mijin85cho your suggestion w.r.t to current rephrase is good 
```suggestion
Change the display of toolbar by setting the `FlexDirection` to **row**, `AlignItems` property to **center**, and `JustifyContent` property to **spaceBetween**. It displays buttons and title horizontally with vertical alignment to the center of the toolbar and having equal space between them.

Reply:
Thanks for checking @nawab-samsung. I also think your latest suggestion in your comment is acceptable. 
@pkalota Would you please check and give us your opinion?

Reply:
I think that whole sentence looks better now and it is easier to understand this subpoint. I will apply it in next push.